packages/icons/src/types.ts

```diff
@@ -0,0 +1,3 @@
+import { CryptoIconName, EthereumTokenIconName, FlagIconName, IconName } from './icons';
+
+export type IconNames = IconName | CryptoIconName | EthereumTokenIconName | FlagIconName;
```

Comment:
Don't name it using plural since this type represent single name in the end (as all unions do). So it should be just `IconName`

Reply:
https://github.com/trezor/trezor-suite/pull/7934/commits/0e6d215c95b3b89b30d3d7cb7b57823b1c38211b